### ISSUE TITLE
aria-labelledby is present, but still erroring and now aria attribute not used correctly

### ISSUE THREAD

**Issue by portermichael** [dequelabs/react-axe#37]
Hi,
I've got a form that I'm trying to attach aria-labelledby to. The error was fixed, but react-axe is still calling an error. I'm also getting the new error that ARIA attributes should be used correctly. I'm guessing it has to do with the hyphen being used in the aria-labelledby. Maybe inconsistent space/kebab space usage? Not ideal, but I didn't think it was rule breaking.

<img width="838" alt="screen shot 2018-05-08 at 5 45 42 pm" src="https://user-images.githubusercontent.com/18373736/39789970-e8ae6354-52e7-11e8-8447-9751a2fb7fd4.png">

I might be making a mistake, but I don't see it. So I thought I'd share it.

Best,
Mike

PS if you can make this title better, please do!

**Comment by WilcoFiers** [comment]
Hey Mike. It looks like you're not using aria-labelledby correctly. Firstly, IDs can't have spaces in them. It's pretty much the only thing not allowed in IDs. Secondly, aria-labelledby should be used to reference a label. You are now having it point to two elements. One with `id="Square"` and a second with `id=Feet-min"`. I'm guessing both of these don't exist?

**Comment by portermichael** [comment]
You are 100% right. Thanks for your help!
